apps/calendar/js/views/settings.js

```diff
@@ -348,59 +291,34 @@ Settings.prototype = {
 
   onactive: function() {
     debug('Will do settings animation.');
-
-    // If we haven't yet cached idb calendars, do that now.
-    var fetch;
```

Comment:
We no longer need to fetch calendars ourselves. Just listen to `calendarObserver`